cmd/podman/images.go

```diff
@@ -92,68 +90,67 @@ func imagesCmd(c *cli.Context) error {
 		return errors.Wrapf(err, "Could not get runtime")
 	}
 	defer runtime.Shutdown(false)
-
-	format := genImagesFormat(c.String("format"), c.Bool("quiet"), c.Bool("noheading"), c.Bool("digests"))
-
-	opts := imagesOptions{
-		quiet:     c.Bool("quiet"),
-		noHeading: c.Bool("noheading"),
-		noTrunc:   c.Bool("no-trunc"),
-		digests:   c.Bool("digests"),
-		format:    format,
-	}
-
+	var filterFuncs []libpod.ImageResultFilter
 	var imageInput string
 	if len(c.Args()) == 1 {
 		imageInput = c.Args().Get(0)
+		//image := runtime.NewImage(imageInput)
```

Comment:
Is this lingering from earlier work?

Reply:
yeah, probably going to find several of those

Reply:
fixed